db/migrate/20160218130351_add_donation_to_actions.rb

```diff
@@ -0,0 +1,5 @@
+class AddDonationToActions < ActiveRecord::Migration
+  def change
+    add_column :actions, :donation, :boolean, default: false
```

Comment:
to me it would be cool if actions had an enumerable column for action_type, with `donation`, `signature`, and then we could add stuff to it (eg `call`) but I reckon we can cross that bridge when we come to it.